src/rt64lib/shaders/LuminanceHistogramCS.hlsl

```diff
@@ -0,0 +1,55 @@
+/*
+*   
+*   RT64
+*   Luminance Histogram by Alex Tardif
+*   From http://www.alextardif.com/HistogramLuminance.html
+*   
+*/
+
+#define NUM_HISTOGRAM_BINS 64
+#define HISTOGRAM_THREADS_PER_DIMENSION 8
+#define EPSILON	1e-6
+
+Texture2D<float3> HDRTexture : register(t0);
+RWByteAddressBuffer LuminanceHistogram : register(u0);
+
+cbuffer LuminanceHistogramBuffer : register(b0)
+{
+	uint inputWidth;
+	uint inputHeight;
+	float minLogLuminance;
+	float oneOverLogLuminanceRange;
+};
+
+groupshared uint HistogramShared[NUM_HISTOGRAM_BINS];
+
+float GetLuminance(float3 color) {
+	return dot(color, float3(0.2127f, 0.7152f, 0.0722f));
+}
+
+uint HDRToHistogramBin(float3 hdrColor)
+{
+	float luminance = GetLuminance(hdrColor);
+	if (luminance < EPSILON) {
+		return 0;
+	}
+	
+    float logLuminance = saturate((luminance - minLogLuminance) * oneOverLogLuminanceRange);
```

Comment:
Since the histogram no longer uses log scale, we'll need to rename the variables accordingly in the constants, shader and cpp side.